TUTORIAL.md

```diff
@@ -377,7 +377,7 @@ Additionally, there is a section explaining how most dispatching bindings allow
 
 ### One-way bindings
 
-*Relevant sample: SingleCounter*
+*Relevant sample: SingleCounter - ([C# wrapper](https://github.com/elmish/Elmish.WPF/tree/master/src/Samples/SingleCounter) which calls [F# core](https://github.com/elmish/Elmish.WPF/tree/master/src/Samples/SingleCounter.Core))*
```

Comment:
Adjust your links so they have this format:

> XAML views and F# core

with the links on the obvious words.